lib/models/villager_model.dart

```diff
@@ -0,0 +1,42 @@
+class villager {
+  String? name;
+  String? imageUrl;
+  String? species;
+  String? personality;
+  String? gender;
+  String? birthdayMonth;
+  String? birthdayDay;
```

Comment:
is everything supposed to be nullable? can the `villager` object itself be nullable?